.github/workflows/ci.yml

```
@@ -37,7 +37,7 @@ jobs:
         run: pip install .
 
       - name: Run tests
-        run: python -m pytest -v --timeout=1800 --durations=20 test
+        run: python -m pytest -v --timeout=1800 --durations=20 test -k test_runner
```

This is a temporary change to fail fast when the `mem` benchmark doesn't work.